boards/stm32f469_discovery/src/sdcard.adb

```diff
@@ -215,7 +224,9 @@ package body SDCard is
       end if;
 
       Ret := STM32.SDMMC.Initialize
-        (Controller.Device.all, Controller.Info);
+        (Controller.Device.all,
+         System.BB.Board_Parameters.Main_Clock_Frequency,
```

Comment:
You should use STM32.Device.System_Clock_Frequencies.SYSCLK. That's what we use all across the library and it will remove the run-time dependency with the warnings off pragma.